services/galley/src/Galley/API/Teams.hs

```diff
@@ -1027,7 +1027,7 @@ deleteTeamMember lusr zcon tid remove mBody = do
   if team ^. teamBinding == Binding && isJust targetMember
     then do
       body <- mBody & note (InvalidPayload "missing request body")
-      ensureReAuthorised (tUnqualified lusr) (body ^. tmdAuthPassword)
+      ensureReAuthorised (tUnqualified lusr) (body ^. tmdAuthPassword) Nothing Nothing
```

Comment:
I wonder if we should add this case, too?  Can you open a jira ticket in "ready to refine"?